tests/generic_config_updater/test_mgmt_interface.py

```diff
@@ -37,11 +37,25 @@ def ensure_dut_readiness(duthost):
 
 def update_forced_mgmt_route(duthost, interface_address, interface_key, routes):
     # Escape '/' in interface key
+    path = (
+        create_path(
+            [
+                "localhost",
+                "MGMT_INTERFACE",
+                "eth0|{}".format(interface_address),
+                "forced_mgmt_routes"
+            ]
+        ) if duthost.is_multi_asic else create_path(
+            [
+                "MGMT_INTERFACE",
+                "eth0|{}".format(interface_address),
+                "forced_mgmt_routes"
+            ]
+        )
+    )
```

Comment:
No need for wrapping the namespace, format_json_patch_for_multiasic will do wrap. Can you please check other test PRs if they are dup?